worker/pdf/jpg_to_pdf.py

```diff
@@ -0,0 +1,31 @@
+import PIL
+import PyPDF2
+
+
+def jpg_to_pdf(source_file, target_file):
```

Comment:
Noch eine Kleinigkeit: Aus Gründen der Konsistenz und der besseren Lesbarkeit sollten Funktionsnamen mit Verben anfangen, also z.B. `convert_jpg_to_pdf` (siehe https://medium.com/mindorks/meaningful-names-a-dimension-of-writing-clean-code-fdae1ae4f0b1).

Reply:
Das gilt in noch höherem Maße für die Task-Namen. So eine Job-Config YAML ist ja auch eine domain specific language und sollte in sich konsistent sein.